testsuite/tests/typing-poly/poly.ml

```diff
@@ -21,6 +21,34 @@ val f : 'a list -> 'a fold = <fun>
 - : int = 6
 |}];;
 
+type pty = {pv : 'a. 'a list};;
+[%%expect {|
+type pty = { pv : 'a. 'a list; }
+|}];;
```

Comment:
Sorry for the minor/superficial question, but why are you using an `expect` test here and for the phrase below? I would only expect tests (heh) on the phrases that do have an interesting behavior, here the `match` one, and the `fun` one used as a counterpart.

Reply:
No, you should view expect blocks as `;;` in the toplevel. You don't have to put it between every phrase, but when you put one, the compiler will process (and output stuff for) everything that precedes it.

Reply:
Obviously I had forgotten this suboptimal constraint, sorry for the noise.

Maybe we could have an `all/last` modifier `[%expect all {|...|}]`, `[%expect last {|...|}]`, such that `all` is the current default and `last` skips the output of all the phrases but the last one.

Reply:
I'm not sure I'd be in favor: it seems good to me to always have the whole output in tests.
But others might disagree.